src/main/java/pl/edu/agh/firecell/engine/algorithm/FirePropagator.java

```diff
@@ -20,13 +20,17 @@ public boolean computeNewFlammable(Cell oldCell, int newBurningTime) {
         return switch (oldCell.material()) {
             case WOOD -> newBurningTime < MAX_BURNING_TIME;
             case AIR -> true;
+            case CELLULAR_CONCRETE -> false;
+            default -> throw new IllegalStateException("Unexpected value: " + oldCell.material());
```

Comment:
No need for default, since we are switching Enum. Java lets us not use default, and we should always do it